observation_portal/requestgroups/test/test_api.py

```diff
@@ -112,13 +113,84 @@ def test_get_request_group_list_authenticated(self):
         result = self.client.get(reverse('api:request_groups-list'))
         self.assertContains(result, request_group.name)
 
-    def test_get_request_group_list_staff(self):
+    def test_get_request_group_list_is_staff_without_staff_view(self):
```

Comment:
The new tests in this file are ones that I pulled over from `observation_portal/requestgroups/test/test_views.py` and refactored to test the requestgroups API endpoints.